File: multiplesOf3and5.m
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% octave multiplesOf3and5.m
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

multiplesOf3 = [];
multiplesOf5 = [];

for i = 1:999
  if (mod(i,3)== 0)
    multiplesOf3 = [multiplesOf3 i];
    %disp(i)
  endif
  if (mod(i,5)== 0)
    multiplesOf5 = [multiplesOf5 i];
  endif
endfor

all = union(multiplesOf3, multiplesOf5);

x = 0;
for i = all
  %disp(i);
  x = x + i;
endfor

disp(x);

File: multiplesOf3and5.py

multiplesOf3 = set();
multiplesOf5 = set();
#multiplesOf3 = {3,6};
#multiplesOf5 = {5,10};
all = {};
sum = 0;

for i in range(1000):
   if (i % 3 == 0):
     multiplesOf3.add(i)
   if (i % 5 == 0):
     multiplesOf5.add(i)

all = multiplesOf3.union(multiplesOf5);

for i in all:
  sum = sum + i;
  #print(i);

print(sum);
